src/Network/Network/help/Get-AzRouteServerPeerAdvertisedRoute.md

```diff
@@ -0,0 +1,144 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/get-azrouteserverpeeradvertisedroute
```

Comment:
We are now using locale-free links which can redirect to the correct language for the viewer
```suggestion
online version: https://docs.microsoft.com/powershell/module/az.network/get-azrouteserverpeeradvertisedroute
```